### PR TITLE
Minor stat buff for shunned races + stat buff for being inhumen gods worshipper

### PR DESCRIPTION
**Pull request by DelineFortune**
## About The Pull Request

Shunned races are minority now as punching bags. They lost their bad reputation because people left the server / started to play respected races for combat roles

Templars are also allowed to be for shunned races because technically they proved themselves as servants of Ten and they are just tools in church's hands (not as acolytes or priest) 

And maybe an opportunity for heretics to use shunned race templars to defile the church

## Why It's Good For The Game

More content, more action in rounds, more ways to play as antag against bored 100500 knights/templars/grenzels/riders/maas/watchmen that can charge-stun and kill your kobold/tiefling
Also more reasons to be racist to shunned races, fear them, call reinforcements (and involve more players into situation,  not 1 vs 1 fight)
Grim dark experience to inquisitors because everyone can be a heretic for blessing of dark gods and try to kill you. 
Templars are servants of Ten, but they can be members of shunned races and help their kins against you. 

Also gives a positive reason to play unhumen worshipper instead of cons that they have: job lock and a reason to be killed for their faith

### CODE REVIEW

**Comment by DelineFortune** [comment]
image

**Comment by Elli-Skala** [comment]
> Also gives a positive reason to play unhumen worshipper instead of cons that they have: job lock and a reason to be killed for their faith
 
 this was a small note how ever what this ment was that the inhuman patrons now give straight stat buffs 
 Zizo - Int +2 Speed +1
Graggar - Str +1 Con +1
Matthios - Spd +1 Per +1
Baotha - Edu + 2

**Comment by UntoldTactics** [comment]
What's stopping someone just being a heretic follower for the stats but not actually doing anything in line with the god they follow? I can see court magis going zizo follower for ez free int lmao.

**Comment by DelineFortune** [comment]
Being heretic is literally frag free reason for others if they have proofs that you are
The mage is zizoid for stats and you dont like them?
Good.
Ask the dungeonner or inquisitor to torture them, if they are - you have a reason to deal with them.

**Comment by john-pawson** [comment]
> Player quality issue, people are picking knights/grenzels/templars to have combat stats and play as security officers of station 13 but they are not following oaths/guild standards/preists orders

Unrelated with what he asked, also templars and knights are known to be punished by directly disobeying and/or not following orders from their leaders, and grenzels are mercs, of course they can do whatever they want as long as they're making sure the guild doesn't hear any complains, might even make interesting conflict if other grenzels stop the unruly one to keep face, it fits.

> Enjoy medium roleplay as it supposed to be

Not an excuse to make bad PRs. 

> Being heretic is literally frag free reason for others if they have proofs that you are The mage is zizoid for stats and you dont like them? 
> Good. 
> Ask the dungeonner or inquisitor to torture them, if they are - you have a reason to deal with them.

I think the problem is that this applies to everyone, literally anyone can go ahead and pick an inhumen god for better stats as long as they're not a keep or church role, and at that point why wouldn't you? It gives no clues about you following an inhumen god, and if the only way for someone to know is to get the inquisitor to interrogate them, why would that ever happen to your average commoner or adventurer? 
Why wouldn't every single advenuter or towner be a Graggarite if it just makes them stronger? And why or how would said person even get found out as a graggarite? Why would they EVER tell to someone they're a graggarite OR get interrogated by a dungeoneer or inquisitor?? It's got little risk.
At some point even people that do actually RP will start to pick inhumen gods because of the just objectively better stats for absolutely no cost. When was the last time anyone got called out for being a zizo follower that wasn't screaming "i'm a zizo follower" or was an actual antag, like lich or bandit?
TL;DR too little risk for too much reward

This also indirectly buffs bandits, lichs and vampires, as they automatically follow inhumen gods like zizo and matthios.

This is just free stats. And a bad idea. If you want it to be a problem make them a known heretic, or give them some kind of obvious flaw that comes from worshipping a dark god, here's a few examples of what you _could_ do so it's not completely free:

- Zizoids might have a dark aura around them that can be detected by mages and/or church people with a high enough arcane/holy skill, like how mages and skilled clerics can detect who's enthralling them or if they're being watched with magic.

- Graggarites might need to eat organs or other disgusting meals to please their appetite or else they start showing signs of being a graggarite.

- Matthites should be as simple as making them a bandit, maybe only known by specific roles like sheriff or marshal to make it different than the common bandit, to show that they're still a bandit, just a bit more hidden and less known.

- Baothans might start to look a bit sickly and lose strength or other stats with very intense withdrawal if they're not high for a long enough time, making taking drugs like eating food or drinking water.


On a different note, don't think having dark elves, created and taught by zizo, or tieflings, corrupted with unholy forces, would make any sense as templars. The main reason why they're shunned is religious and because of the chruch in the first place.

>More content, more action in rounds, more ways to play as antag against bored 100500 knights/templars/grenzels/riders/maas/watchmen that can charge-stun and kill your kobold/tiefling
> Also more reasons to be racist to shunned races, fear them, call reinforcements (and involve more players into situation, not 1 vs 1 fight)

- This doesn't add content or action in rounds: As most people that do pick inhumen gods will never be found out. 
- It doesn't add ways to play as an antag: As it doesn't affect antags rather than indirectly buffing them, unless you count anyone that chooses an inhumen patron as an antag, in which case you're just wrong. 
- And doesn't add more reasons to be racist against shunned races: as shunned races don't have any patron locks, and this update only changes them being allowed to be templars, which is unrelated to them being feared. This might even take reasons to be racist towards them, since now apparently the church is OK with having them as templars it'd mean they're getting included more.

**Comment by DorfGuy1** [comment]
while this is fun, people are totally going to pick them just for the combat gigastats. Graggar orc is gonna go crazy. Also someone mentioned guild code, in respect to merecenaries, but i've never seen it written down anywhere. It's a wonder that Grenzles don't betray their in universe reputation more as is. What this is gonna mean is that the church is gonna fight the keep more cause 'they stole our buddy (who's a totally ass who's never done a damn thing for any of us) lets fight the church'. This is what happens every time one of the Men at arms or Watch decide that they're going to cause problems for no reason and act like a common criminal( if you act like that, how'd you get hired for that?). Nobody gives a shit that the religious authority is calling out Joe the grabby guardsman. It's gonna get worse with this, because not only will joe the grabby guardsman be doing the same old stuff, he will have better stats too. And a possible excuse to screw over the church.

**Comment by Cyprex** [comment]
The follower buffs are definitely too much- the other changes we can *maybe* talk about.

**Comment by radarseas** [comment]
I don't like stat increases for clicking a god. If you want a god buff, do it the interesting way, like how graggar lets you eat raw flesh, which both is a dead giveaway but also a pretty good buff at the same time.

Matthios is fine as is, knowing fellow followers can turn the tides in the least expected way like 4 matthiosians suddenly jumping a vigilantee with the only giveway being a few examines and nods.

Baothia's buff could use more honestly, make it give you another stat increasing buff instead for the more drugs in your system at once, to encourage binging and variety.

Zizoiods are the only ones in need of a unique blessing. All things considered its hard to really think of a buff for them.

**Comment by NikolayByondPlayer** [comment]
Shunned should suffer, what is the point of being shunned then if you can kill everything

**Comment by Hmeister-real** [comment]
> Shunned should suffer, what is the point of being shunned then if you can kill everything

to be a minor antagonist because there is a major lack of action in the server
